app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Recipe App</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+    <nav>
+      <%= link_to 'Recipe App', root_path, class: 'logo' %>
+      <div>
+        <% if user_signed_in? %>
+          <%= link_to 'Public Recipes', public_recipes_path %>
+          <%= link_to 'My Recipes', recipes_path %>
+          <%= link_to 'New Recipe', new_recipe_path %>
+          <%= link_to 'Foods', foods_path %>
+          <%= link_to 'Shopping List', shopping_list_path %>
+        <% else %>
+          <%= link_to 'Public Recipes', public_recipes_path %>
+        <% end %>
+      </div>
+      <div>
+        <% if user_signed_in? %>
```

Comment:
#### Optional

 -  [ ] To improve user experience, it's advisable to make the navigation menu more user-friendly and understandable. For instance, the "sign out" button seems incomplete, and "profile out" might not be clear to users. However, the addition of "profile settings" functionality was commendable! 🚀👍

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/4bf6ae1a-f12c-4f74-ae1f-37317989a700)
 
#### Keep up the good work 👏

Reply:
- Well resolved